### ISSUE TITLE
Uncaught Error: Class 'Automattic\Jetpack\Connection\Client' not found

### ISSUE THREAD

**Issue by bobbingwide** [Automattic/jetpack#13396]
The following error was displayed after updating to the latest version of Jetpack.
```
Fatal error: Uncaught Error: Class 'Automattic\Jetpack\Connection\Client' not found in /home/bobbingw/public_html/wp-content/plugins/jetpack/class.jetpack-ixr-client.php:38 


Stack trace:
--
  | #0 /home/bobbingw/public_html/wp-content/plugins/jetpack/sync/class.jetpack-sync-actions.php(191): Jetpack_IXR_Client-&gt;query('jetpack.syncAct...', Array)
  | #1 /home/bobbingw/public_html/wp-includes/class-wp-hook.php(286): Jetpack_Sync_Actions::send_data(Array, 'deflate-json-ar...', 1567503609.3813, 'sync', 0.0021119117736816, 0.0080699920654297)
  | #2 /home/bobbingw/public_html/wp-includes/plugin.php(208): WP_Hook-&gt;apply_filters(Array, Array)
  | #3 /home/bobbingw/public_html/wp-content/plugins/jetpack/sync/class.jetpack-sync-sender.php(244): apply_filters('jetpack_sync_se...', Array, 'deflate-json-ar...', 1567503609.3813, 'sync', 0.0021119117736816, 0.0080699920654297)
  | #4 /home/bobbingw/public_html/wp-content/plugins/jetpack/sync/class.jetpack-sync-sender.php(124): Jetpack_Sync_Sender-&gt;do_sync_for_queue(Object(Jetpack_Sync_Queue))
  | #5 /home/bobbing in <b>/home/bobbingw/public_html/wp-content/plugins/jetpack/class.jetpack-ixr-client.php</b> on line <b>38</b><br />

```

#### Steps to reproduce the issue

1. Go to 'Update Plugins'
2. Click on selected plugins to update.
3. Perform update.

#### What I expected

Updates and No error.

#### What happened instead
Jetpack was successfully updated to 7.6 but the Fatal error was displayed at the end of the page. 
I have not tried to reproduce the problem anywhere else. 

**Screenshots**
![image](https://user-images.githubusercontent.com/2474435/64162898-1954f780-ce38-11e9-9f9c-c26250fc7fda.png)

**Comment by jeherve** [comment]
Can you reproduce with Jetpack 7.7, scheduled to be released in a few hours?
https://github.com/Automattic/jetpack/releases

Could you also try to reproduce on a fresh site with no other plugins? Do you experience similar issues then?

**Comment by danielbachhuber** [comment]
I _think_ we just saw a similar issue going from Jetpack 7.6 to 7.7.1.

We use Staging Pilot to do auto-upgrades. WordPress' PHP error detection noticed an error right about the time Staging Pilot was running:

```
An error of type E_ERROR was caused in line 69 of the file /nas/content/live/pinchofyum/wp-content/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php. Error message: Uncaught Error: Class 'Jetpack_IXR_Client' not found in /nas/content/live/pinchofyum/wp-content/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php:69
Stack trace:
#0 [internal function]: Jetpack_Core_Json_Api_Endpoints::register_endpoints(Object(WP_REST_Server))
#1 /nas/content/live/pinchofyum/wp-includes/class-wp-hook.php(286): call_user_func_array(Array, Array)
#2 /nas/content/live/pinchofyum/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#3 /nas/content/live/pinchofyum/wp-includes/plugin.php(465): WP_Hook->do_action(Array)
#4 /nas/content/live/pinchofyum/wp-includes/rest-api.php(475): do_action('rest_api_init', Object(WP_REST_Server))
#5 /nas/content/live/pinchofyum/wp-content/plugins/wpe-advanced-cache-options/inc/wpeac-admin.php(526): rest_get_server()
#6 /nas/content/live/pinchofyum/wp-content/plugins/wpe-advanced-cache-options/inc/wpeac-admin.php(323): WPEAC_Admin->get_rest_api_namespaces()
#7 [internal function]: WPEAC_Admin->register_settings('')
#8 /nas/content/live/
```

**Comment by danielbachhuber** [comment]
Actually, I think my issue was fixed by #13422

**Comment by jeherve** [comment]
Yes, I believe the issue reported above was also fixed by that PR, hence my question earlier. I'll close this now. Feel free to comment or reopen if needed!
